internal/provider/resource_user_test.go

```diff
@@ -32,15 +34,16 @@ func TestAccPritunlUser(t *testing.T) {
 	})
 }
 
-func testPritunlUserConfig(username, orgName string) string {
+func testPritunlUserConfig(username, orgName, pin string) string {
```

Comment:
Create an additional config helper and a new test case instead of modifying this one, because pin is an optional value for a user, and it's better to have a new test case for this scenario.

Reply:
Added a new test case and helper function.